src/views/account/Lessons/Lessons.tsx

```diff
@@ -43,6 +45,9 @@ const Lessons: Screen<"Lessons"> = ({ route, navigation }) => {
   const loadedWeeks = useRef<Set<number>>(new Set());
   const currentlyLoadingWeeks = useRef<Set<number>>(new Set());
 
+  const [shouldShowWeekFrequency, setShouldShowWeekFrequency] = useState(false);
```

Comment:
j'aurai préféré que ça soit activé par défaut
```suggestion
  const [shouldShowWeekFrequency, setShouldShowWeekFrequency] = useState(true);
```

Reply:
Fait 👍